classes/person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
   end
 
```

Comment:
- [ ] A method for adding rentals should be present. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please kindly add the method, it can be similar to 👇:

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
thanks a lot.. fix all these changes..